lang/golang/golang/Makefile

```diff
@@ -251,8 +274,13 @@ define Host/Compile
 		$(HOST_GO_VARS) \
 	)
 
-	$(call GoCompiler/Host/Make, \
+	$(call GoCompiler/Bootstrap-1.21/Make, \
 		GOROOT_BOOTSTRAP="$(BOOTSTRAP_1_17_BUILD_DIR)" \
```

Comment:
see the comments below (https://github.com/openwrt/packages/pull/23475#discussion_r1518812323)

Reply:
I think it does not help me. You call Bootstrap 1.21 and then the goroot buildir is 1.17. That does not seems to be intentional or is it? Still all Go lang packages requirest up to date Go, so having three bootstrap does not seems right. Why do we need to keep go1.4-bootstrap-20171003.tar.gz?

Reply:
This is intentional. The `GOROOT_BOOTSTRAP` is for specifying the previous built go toolchain, to bootstrap the next stage.

go1.4 is required for bootstrapping because it's the last version of go with its compiler written in C.

Reply:
Oh, you are right. I learned and checked how the Buildroot does it and there are those two stages. Seems complicated, anyway thanks for your time!